duqtools/jetto/_llcmd.py

```diff
@@ -0,0 +1,34 @@
+"""Function to create llcmd file."""
+from duqtools.config.workdir import WorkDirectory
```

Comment:
We are using `WorkDirectory` only for type checking, so we can do this to avoid circular dependencies. In that case, we can keep the `__init__` imports in this directory.

```suggestion
from __future__ import annotations

from typing import TYPE_CHECKING

if TYPE_CHECKING:
    from duqtools.config.workdir import WorkDirectory
```